deku-p/src/core/crypto/tests/tezos_test_data.ml

```diff
@@ -0,0 +1,14 @@
+module type Tezos_data = sig
+  val public_keys : string list
+end
+
+module Ed25519_data : Tezos_data = struct
+  let public_keys =
```

Comment:
You can add a comment showing which line you executed on `dune utop`.

Or alternatively you can just add a dependency to tezos-crypto and use here directly, instead of hard coding the cases, this would be better as it would also detect changes to tezos-crypto, but can be done as a future improvement.

Reply:
The tezos-crypto data generation was done through a script that requires `tezos-crypto` to run. Until we can have it available on Deku as a dev dependency, the script will be included as a .txt file.

Once all the data is included and `tezos-crypto` can be used on Deku, the generated data will be replaced by direct comparisons between the output of `tezos-crypto` and `deku-crypto`.